contents/handbook/help.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: How you can help
+sidebar: Handbook
+showTitle: true
+---
+
+## Getting yourself up and running - your first week
+
+If you're new, the default goal is to be able to get work done autonomously.
+
+This will require:
+
+* You know how to ship things in our environment and with your equipment
+* You get to know your team and have a sense of who can QA your work
+* You get what the company (and your small team) care about and needs to get done, so you can prioritize appropriately
+
+Everything else is a means to this end! We often do onboarding in person to accelerate all of the above. This usually takes around a week.
+
+## General etiquette
+
+### Don't assign issues to people
+
+You can list and categorize issues.
+
+If you want someone to see an issue, @mention them and/or Slack them the link.
+
+### Don't yolo merge
+
+Do not "yolo merge", ie: force a change to our website or platform without someone else checking it. This should _only_ happen in emergencies, _even_ for simple changes. It is _so_ frequent that we find issues. If you have _any_ doubt, get someone else to look at it first.
+
+### PRs > issues > Slack
+
+Bias for impact. If you can just pick up the work, do so. We want a culture of individual contribution _not_ of delegation.
+
+It is fine (and encouraged) to pick up side quests or to deviate from your goals if you think you should. Especially if something is a quick fix, do it yourself as part of our value that _Everyone Codes_.
+
+If you aren't able to make a change yourself, create an issue in GitHub. Avoid simply relaying todos in Slack as a means of getting someone to pick up a task. It's hard to track and easy to forget.
+
+### Do things as publicly as possible by default
+
+For discussions, public repos are the best place. Then private ones, then Slack public channels, then Slack private channels or DMs. This is part of our _"We are open source"_ value, and helps with general context setting for the wider team, which means everyone can work more autonomously.
+
+There are only a few exceptions to what we can't share publicly, for example if you are discussing security concerns, specific customers (for privacy reasons), revenue or growth numbers (since these cause signalling issues with investors or competitors).
+
+Internally, _everything_ can be shared apart from people issues - such as HR / personal (ie: recruitment or health data).
+
+### Be proactive with community questions
+
+Don't _only_ help the community when you're the person on support hero in your small team. No matter what your goals may be, if you can quickly ship fixes to real life user problems, then you are going to build goodwill, word of mouth growth, and a better product all in one swoop.
+
+You can find these in [posthog.com/questions](https://posthog.com/questions).
+
+## Longer term - values
```

Comment:
I think this deserves a separate chapter

Reply:
done